source/_integrations/ialarm.markdown

```diff
@@ -4,6 +4,7 @@ description: Instructions on how to integrate iAlarms alarms into Home Assistant
 logo: antifurto365-ialarm.png
 ha_category:
   - Alarm
+ha_iot_class: Local Push
```

Comment:
It uses polling to get it. This platform component uses the default scan interval and default polling behavior.

```suggestion
ha_iot_class: Local Polling
```